web/src/components/demo_pages/Intro/intro.css

```diff
@@ -0,0 +1,91 @@
+@import url('../../vars.css');
+
+.container {
+    height: 100vh;
+    width: 100vw;
+    padding-top: 2.5vh;
+    background: linear-gradient(180deg, #f3f2f1 0%, #ffffff 101.23%);
+}
+
+.text-box {
+    display: flex;
+    flex-direction: column;
+    padding: 0px;
+    height: 173px;
+    left: calc(50% - 700px / 2 + 1px);
+    bottom: 68px;
+}
+
+.text > h1 {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
+    line-height: 43px;
+    text-align: center;
+}
+
+.text > p {
+    font-style: normal;
+    font-weight: 300;
+    font-size: 24px;
+    line-height: 33px;
```

Comment:
Similarly to my above comments, the font-size here is specified in `vars`, and for the sake of accommodating people who may change their browser's font-size settings it's always a good idea to specify things like line-height in relative/unitless terms (e.g. `1.375` in this case, although if you wanted to call it `1.4` I also would not have any objections.) 

More info on unitless here: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height